modules/administration/pages/ssl-certs.adoc

```diff
@@ -36,20 +36,28 @@ If you are using intermediate CAs in a chained certificate, the root CA is insta
 During the SSL handshake, clients must be able to verify the entire chain of intermediate certificates between the root CA and the server certificate, so they must be able to access all the intermediate certificates.
 
 There are two main ways of achieving this.
-In {productname}, by default, all the intermediate CAs are installed on the client.
+In older versions of {productname}, by default, all the intermediate CAs are installed on the client.
 However, you could also configure your services on the server to provide them to the client.
 In this case, during the SSL handshake, the server presents the server certificate as well as all the intermediate CAs.
+This mechanims is used now as the new default configuration.
 
-Whichever method you choose, you must ensure that the ``CA_CERT`` environment variable points to the root CA, and all intermediate CAs.
-It should not contain the server certificate.
-The server certificate must be defined at the ``SERVER_CERT`` environment variable.
+//Whichever method you choose, you must ensure that the ``CA_CERT`` environment variable points to the root CA, and all intermediate CAs.
+//It should not contain the server certificate.
+//The server certificate must be defined at the ``SERVER_CERT`` environment variable.
 
-By default, {productname} uses a self-signed certificate.
+By default, {productname} uses a self-signed certificate without intermediate CAs.
 For additional security, you can arrange a third party CA to sign your certificates.
 Third party CAs perform checks to ensure that the information contained in the certificate is correct.
 They usually charge an annual fee for this service.
 Using a third party CA makes certificates harder to spoof, and provides additional protection for your installation.
 If you have certificates signed by a third party CA, you can import them to your {productname} installation.
 
```

Comment:
// FIXME: I must check the following paras when this PR is ready for review. -ke-